exo-go/src/cmd/fetch-templates.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"path"
+
+	"github.com/Originate/exosphere/exo-go/src/app_config_helpers"
+	"github.com/Originate/exosphere/exo-go/src/os_helpers"
+	"github.com/Originate/exosphere/exo-go/src/template_helpers"
+	"github.com/Originate/exosphere/exo-go/src/util"
+	"github.com/spf13/cobra"
+)
+
+var fetchTemplateCmd = &cobra.Command{
+	Use:   "fetch-templates",
+	Short: "Fetch service templates",
+	Long: `Fetch service templates
+This command must be called in the root directory of the application`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if util.PrintHelpIfNecessary(cmd, args) {
+			return
+		}
+		fmt.Print("We are about to fetch remote templates\n\n")
+		appConfig := appConfigHelpers.GetAppConfig()
+		if len(appConfig.Templates) > 0 {
```

Comment:
Is this if-clause really necessary? The for loop just does nothing if appConfig.Templates is an empty array.